docs/source/overview/delivery_services.rst

```diff
@@ -0,0 +1,755 @@
+	| totalKbpsThreshold | In :ref:`to-api` responses - most notably :ref:`to-api-cdns-name-configs-monitoring` | unchanged (numeric), but converted from :abbr:`Mbps (Megabits per second)` to :abbr:`Kbps (kilobits per second)` |
+	+--------------------+--------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------+
+
+Global Max TPS
+--------------
+The maximum :abbr:`TPS (Transactions per Second)` this Delivery Service can serve across all :term:`Edge-tier cache server`\ s before traffic will be diverted to the bypass destination. For a DNS-:ref:`Routed <ds-types>` Delivery Service, the `DNS Bypass IP`_ or `DNS Bypass IPv6`_ will be used (depending on whether this was a A or AAAA request), and for HTTP-:ref:`Routed <ds-types>` Delivery Services the `HTTP Bypass FQDN`_ will be used.
+
+.. table:: Aliases
+
+	+-------------------+--------------------------------------------------------------------------------------+---------------------+
+	| Name              | Use(s)                                                                               | Type(s)             |
+	+===================+======================================================================================+=====================+
+	| totalTpsThreshold | In :ref:`to-api` responses - most notably :ref:`to-api-cdns-name-configs-monitoring` | unchanged (numeric) |
+	+-------------------+--------------------------------------------------------------------------------------+---------------------+
+
+HTTP Bypass FQDN
+----------------
+When the limits placed on this Delivery Service by the `Global Max Mbps`_ and/or `Global Max Tps`_ are exceeded, an HTTP-:ref:`Routed <ds-types>` Delivery Service will direct excess traffic to this :abbr:`Fully Qualified Domain Name`.
+
+IPv6 Routing Enabled
+--------------------
+A boolean value that controls whether or not clients using IPv6 can be routed to this Delivery Service by Traffic Router. When creating a Delivery Service in Traffic Portal, this will default to "true".
+
+Info URL
+--------
+This should be a URL (though neither the :ref:`to-api` nor the Traffic Ops Database in any way enforce the validity of said URL) to which administrators or others may refer for further information regarding a Delivery Service - e.g. a related JIRA ticket.
+
+Initial Dispersion
+------------------
+The number of :term:`cache servers` between which traffic requesting the same object will be randomly split - meaning that if 4 clients all request the same object (one after another), then if this is above 4 there is a possibility that all 4 are cache misses, necessitating a fresh pull of the content from the next-highest level in the CDN. For most use-cases, this should be ``1``.
```

Comment:
The information about cache misses seems irrelevant to this field. We should not this only applies to HTTP-type delivery services. Similar to the max_dns_answers field, we should just note that it should be scaled up depending on the amount of traffic you expect for assets of this delivery service.

Reply:
I included the "cache misses" stuff because I don't think just that first part is enough to grasp what this is and why it matters. I know for sure it wasn't for me. It's not very well worded, though, and if an improvement can be made I'd love to do it.

Reply:
"The number of edge caches that a particular asset will be distributed across within a cachegroup. For most use-cases, this should be 1, meaning that all clients requesting a particular asset will be directed to a 1 cache. Depending on the popularity and size of assets, you should consider increasing this number in order to spread the request load across more than 1 cache. The larger this number, the more copies of a particular asset are stored in a cachegroup, which pollutes caches (if load distribution is unnecessary) and decreases caching efficiency (due to cache misses if the asset is not requested enough to stay fresh in all the caches)."

How does that sound?